sources/packages/api/src-sql/EducationPrograms/Create-part-time-basis-program.sql

```diff
@@ -0,0 +1,10 @@
+-- Add the column part_time_basis_program for Program with a dummy default value.
+ALTER TABLE sims.education_programs ADD 
+    COLUMN IF NOT EXISTS part_time_basis_program VARCHAR(50) NOT NULL DEFAULT 'no';
+COMMENT ON COLUMN sims.education_programs.part_time_basis_program IS 'If part_time_basis_program is yes, then the program is both Full-Time and Part-Time, if part_time_basis_program is no, then the program is only Full-Time basis';
```

Comment:
Suggestion on column rename: Can we remove basis in part_time_basis_program? it would be better if, we have part_time_program as yes or no.

Reply:
mm, do you find it confusing? it's pretty straight forward right?okay, let me know if you want me to update the field name @guru-aot

Reply:
I felt its quite large, its fine, just a suggestion.

Reply:
Just adding my thoughts also 😄...
I would suggest that the column should be called something like "intensity" and instead of yes/no we can save something like "partTime"/"fullTimePartTime" or whatever makes sense from database/api perspective.
The UI would still be the same with the Yes/No.

Reply:
For the program I think it makes sense that if we have intensity with part time or parttimefulltime. Thoughts?